src/index.html

```diff
@@ -1,12 +1,351 @@
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo logo" href="#">
+        <img
+          src="images/logo.png"
+          alt="Logo"
+          class="logo__image"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="apple">
+              apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="samsung">
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="smartphones">
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="laptops"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="gadgets">
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="tablets">
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="photo">
+              photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="video">
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="cards">
+      <article class="cards__card card" data-qa="card">
+        <img
+          src="/images/imac.jpeg"
+          alt="Imac Photo"
+          class="card__image"
+        >
+
+        <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <p class="card__productCode">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__review-count">Reviews: 5</p>
+        </div>
+
+        <div class="card__price-container">
+          <p class="card__price-label">Price:</p>
+          <p class="card__price">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy"
+          data-qa="card-hover"
+        >BUY
+        </a>
```

Comment:
Keep correct indentation for content.
Also, it's better to make text uppercase in the styles.
```suggestion
        <a
          href="#"
          class="card__buy"
          data-qa="card-hover"
        >
          buy
        </a>
```